src/main/java/at/fhv/ss22/ea/f/musicshop/backend/application/impl/SaleApplicationServiceImpl.java

```diff
@@ -76,6 +77,24 @@ public UUID buy(List<SoundCarrierAmountDTO> carrierAmounts, String paymentMethod
         return saleRepository.saleByInvoiceNumber(invoiceNumber).map(this::saleDtoFromSale);
     }
 
+    @Override
+    public void refund(String invoiceNumber, List<RefundedSaleItemDTO> refundedSaleItems) {
+        //TODO replace with domain specific exceptions
```

Comment:
werden änderungen in domain-objecten tatsächlich an die datenbank weitergegeben? weil i glob dass änderungen wirklich in die datenbank übernommen werden brucht hibernate a transaction

Reply:
Bei Springboot wars mol so dass Hibernate so updates immer vo selber gmacht hot aba döt homma glob @Transactional ket. I probiers no mitm Client us und schau obs des denn automatisch ind Datenbank speichert

Reply:
Hobs usprobiert, es brucht a Transaction. Es müsst jetzt alles mol funktionierta mitm Exchange/Return vo da Sound Carrier